api/Console.json

```diff
@@ -320,7 +320,10 @@
               },
               "edge": {
                 "version_added": "12",
-                "notes": "In EdgeHTML, <code>%c</code> is not supported, and <code>%d</code> outputs a 0 if the specified value isn't a number. This behavior has been fixed in Chromium versions of Edge."
+                "notes": [
+                  "Before Edge 79, <code>%c</code> is not supported.",
```

Comment:
Should these use past tense?

Reply:
I've been in the habit of writing notes in the present (to suggest that, even if a newer version exists, previous versions continue to do whatever they do). But we don't have a style guide for notes and probably ought to. Opened #6229 for this.

Reply:
I thought we were using present tense for a reason, but I don't remember anymore. Thanks for filing the issue, we can address that there and merge this for now.